lib/log.js

```diff
@@ -51,7 +53,16 @@ function log (state, emitter, app, localEmitter) {
     this.name = name
     this.data = data === undefined ? '<no data>' : data
     this.state = shouldDebug
-      ? clone(state)
+      ? tryClone(state)
       : '<disabled>'
   }
+
+  function tryClone (state) {
+    try {
+      return clone(state)
+    } catch (ex) {
+      _log.warn('Could not clone your app state. Make sure to have a serializable state so it can be cloned')
```

Comment:
This will create a huge amount of warnings; could we perhaps emit the warning once when people call `choo.log`? Perhaps we should set a flag that's toggled or something when we notice state cannot be serialized.